cli.js

```diff
@@ -46,8 +46,8 @@ if (command === 'link') {
   var arg2 = argv._[2]
   var arg3 = argv._[3]
 } else {
-  var arg2 = argv._.slice(2).join(' ')
-  var arg3 = null
+  arg2 = argv._.slice(2).join(' ')
```

Comment:
They are declared because JavaScript doesn't have block scope, only function scope, so they get hoisted. However, this might be confusing so I'll fix it. Thanks for noticing it!